### ISSUE TITLE
Redstone Furnace Recipe does not work.

### ISSUE THREAD

**Issue by unpure** [jaredlll08/ModTweaker2#191]
Hey there.
I'm currently trying to fix some issues in my modpack with modtweaker.
For some reasons there seems to be a recipe to smelt draconium dust to draconium ingots but its not possible to place the dust inside of the redstone furnace from TE4. Because of that i tried to fix the problem. I removed the recipe and added a new one, but thats not working.

```
mods.thermalexpansion.Furnace.removeRecipe(<DraconicEvolution:draconiumIngot>);
mods.thermalexpansion.Furnace.addRecipe(1600, <DraconicEvolution:draconiumDust>, <DraconicEvolution:draconiumIngot>);
```

After adding these lines to the script and reloading, modtweaker throws this:

```
WARNING: 1 modifications were stuck
Scripts reloaded
```

Modtweaker added the new recipe and it can be found in NEI but the old broken recipe is still there.
Everytime i reload the script the warning has 1 more midification that is stuck. If i try to remove the recipe and reload the script, it still tells me "* modifications were stuck", but the recipe i added is gone.
Everything else with modtweaker is working fine, only the redstone furnace seems to be glitched.
I dont know if it is a problem with modtweaker or Draconic Evolution that causes this, maybe even some other mods.

A full list of mods that are used in my modpack can be found here:
http://upclan.de/modpack/info/

I'm using ModTweaker2-0.8.3 with forge 10.13.4.1481

Sorry for my bad english, i hope its not to hard to understand.

**Comment by UndeadZeratul** [comment]
Reloading MineTweaker/ModTweaker doesn't work very well when modifying certain recipes.  If you were to restart your minecraft client with the script included, does it still work?

This issue has been known for a while, and until Minetweaker changes its core logic on altering recipe registries, ModTweaker can't do anything about it.

**Comment by unpure** [comment]
I tried to restart the game but its still the same issue. The old broken recipe that should be removed is active in NEI and modtweaker still tells me a modification is stuck.

**Comment by jaredlll08** [comment]
Need the minetweaker log.

**Comment by unpure** [comment]
On world load:
http://pastebin.com/sG5CCEGi

On script reload:
http://pastebin.com/tkW7DkwJ

**Comment by jaredlll08** [comment]
This seems to only affect reloading?

**Comment by unpure** [comment]
Thats almost correct. It only throws the error after reloading, but its still not removing the recipe that should be removed. Its there after loading the world and its there after reloading the scripts.

**Comment by jaredlll08** [comment]
Honestly no idea, but we don't accept issues related to reloading.

**Comment by unpure** [comment]
Thats ok but its not about the reloading.
Its about that the "mods.thermalexpansion.Furnace.removeRecipe" function is not working.
Maybe it is just throwing the error after reloading but it is not removing the recipe.
Thats my problem. I dont care about the error/warning message.

**Comment by Zixxl** [comment]
Please test again with the newest Version of ModTweaker2. It's been recently released as version 0.9.0.

**Comment by unpure** [comment]
I tested everything with modtweaker 0.9.1 and now everything seems to work the way it should.
Thank you for your help.
